src/detect-parse.c

```diff
@@ -1734,6 +1734,80 @@ int DetectSignatureAddTransform(Signature *s, int transform, void *options)
     SCReturnInt(0);
 }
 
+int DetectSignatureSetMultiAppProto(Signature *s, const AppProto *alprotos)
```

Comment:
should add comment about how alprotos is an array terminated by `ALPROTO_UNKNOWN`

Reply:
right